openlibrary/templates/account/reading_log.html

```diff
@@ -24,14 +24,24 @@
 $add_metatag(property="og:description", content=og_description)
 $add_metatag(property="og:image", content=meta_photo_url)
 
+<form method="GET" class="olform pagesearchbox">
+  <input type="text" minlength="3" placeholder="Search by Title or Author" name="q" value="$(query_param('q', ''))"/>
```

Comment:
I think it's fine to allow normal searches here and not limit to just title/author. I think it's kind of a nice super power.

```suggestion
  <input type="text" minlength="3" placeholder="$_('Search your reading log')" name="q" value="$(query_param('q', ''))"/>
```